packages/react-components/lib/item-unknown.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { SimpleInfo } from './simple-info';
+
+interface Location {
+  x: number;
+  y: number;
+  yaw: number;
+  level_name: string;
+}
+
+export interface ItemUnknownProps {
+  name: string;
+  location: Location | undefined;
```

Comment:
There's an issue with requiring context specific props, we can't be sure all consumers will have these props, it is also a problem if a consumers has information it wants to display on unknown state that does not fit into `name` and `location`.

There is also UX problems when the state suddenly change from something unrelated to a table with name, location and error fields.